### ISSUE TITLE
GitHub oauth permission too broad

### ISSUE THREAD

**Issue by drzraf** [IssueHunt/readme#5]
The majority of developers would never let a new website the freedom to fully **act on their behalf** on their GitHub account.
Getting identity should be enough for anyone interest only in public issues/pull-requests.

![Screenshot from 2020-11-27 12-19-02](https://user-images.githubusercontent.com/1169926/100463536-c30e4d00-30aa-11eb-90d9-adbc2e9b8af3.png)

**Comment by theophanemayaud** [comment]
I agree I have this same problem. I am not gonna accept this authorization !

**Comment by psiinon** [comment]
Ditto. This looks really useful, but no way I'll allow it to "Act on my behalf" :(

**Comment by j-tirelli** [comment]
+1, not gonna happen.
